CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to Heroic
+
+Thanks for taking the time to contribute! 
+
+The following is a set of guidelines for contributing to Heroic. 
+
+## Ground Rules
+
+* Be sure to add a comment to your PR explaining the reason for the change. Please include some context behind a bug or issue you ran in to, or your use case and why you need a particular feature added. 
```

Comment:
nitpicky, but I suggest changing `comment` to something like `title and description`.

Reply:
+1 maybe include https://medium.com/@steveamaza/how-to-write-a-proper-git-commit-message-e028865e5791 which dives into writing a "good" commit message